api/src/controllers/common.ts

```diff
@@ -0,0 +1,16 @@
+import { ILogin } from "../models/commonRequest";
+import services from "../services";
+import { Request, Response } from "express";
+import * as response from "../utils/response";
+export const login = async (req: Request, res: Response) => {
```

Comment:
cách trên 1 dòng
- các func cách nhau bởi 1 dòng
- các block code phải các nhau ra bằng 1 dòng để dễ đọc code
- fix lại toàn bộ project

Reply:
done